Exiled.Events/EventArgs/Player/ClosingGeneratorEventArgs.cs

```diff
@@ -30,17 +30,17 @@ public ClosingGeneratorEventArgs(Player player, Scp079Generator generator, bool
         }
 
         /// <summary>
-        ///     Gets or sets a value indicating whether or not the generator can be opened.
+        ///     Gets or sets a value indicating whether or not the generator door can be opened.
         /// </summary>
         public bool IsAllowed { get; set; }
 
         /// <summary>
-        ///     Gets the generator that is opening.
+        ///     Gets the generator door that is opening.
```

Comment:
Feel like this should remain the same, as the class is still `Generator`, it is referencing the generator as a whole, not just the doors.

Reply:
Thats understandable. I have changed that back